src/Castle.Core.Tests/PublicApiTestCase.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright 2004-2021 Castle Project - http://www.castleproject.org/
+﻿// Copyright 2004-2022 Castle Project - http://www.castleproject.org/
```

Comment:
(You could perhaps squash the two "update copyright header" commits into a single one. But I'm fine with it either way.)

Reply:
I tend to favour full history, so haven't had much experience with squashing commits - happy to give it a shot if that's the preference here though.